### ISSUE TITLE
BUG: If I press "reload plugin data", the whole app freezes

### ISSUE THREAD

**Issue by DRYN07** [Flow-Launcher/Flow.Launcher#2844]
### Checks

- [X] I have checked that this issue has not already been reported.

- [X] I am using the latest version of Flow Launcher.

### Problem Description

Just like title says

### To Reproduce

press reload plugin data

### Flow Launcher Version

latest

### Windows Build Number

latest

**Comment by z1nc0r3** [comment]
can you please add a log file?

**Comment by deefrawley** [comment]
Which button are you pressing? Does the same thing happen when you hit F5 when Flow Is active?

**Comment by DRYN07** [comment]
I tried with Enter, click and F5 too. The app does not crash, it is just not react to anything after nor the shortcut, not on the tray icon. I use it as a portable version if it matters.

I waited for a few minutes if the reloading takes longer but nothing happens.

I attach the log and a short video where I tried to refresh with F5

[2024-07-18.txt](https://github.com/user-attachments/files/16280711/2024-07-18.txt)

https://github.com/user-attachments/assets/72484157-abf2-4b56-b774-2b106cfc6227
